src/core/util/typing.py

```diff
@@ -0,0 +1,17 @@
+
+# NOTE(krishan711): waiting for JSON type to be defined in mypy: https://github.com/python/typing/issues/182
+# NOTE(krishan711): copied from https://gist.github.com/BMDan/ede923f733dfdf5ed3f6c9634a3e281f
+from typing import Any, List, Mapping, Union
+JSONV = Union[str, int, float, bool, None]
```

Comment:
omg. this file 😂

Reply:
ha ye they still havent fixed it! Need to get @PattenR to make a legendary PR in mypy

Reply:
Ah you just need TypedDicts, they'll sort a lot of cases where you would normally use these JSON types. I'm still torn on whether JSON types like these a good thing lol